packages/web3-core/src/web3_config.ts

```diff
@@ -22,6 +22,8 @@ export abstract class Web3Config
 		transactionBlockTimeout: 50,
 		transactionConfirmationBlocks: 24,
 		transactionPollingTimeout: 750,
+		transactionReceiptPollingInterval: 1000,
+		transactionConfirmationPollingInterval: 13000,
```

Comment:
This doesn't exist in `1.x` and instead uses `transactionPollingInterval` (which I renamed to `transactionReceiptPollingInterval` in this PR),  but I believe it would be better to split these configurations as `1.x` was requesting a block every 1 second (by default) to check for a new block, while the [average block time](https://ycharts.com/indicators/ethereum_average_block_time) is around 13 seconds